### PR TITLE
Add tree

### PR DESCRIPTION
**Pull request by vokarl**

### CODE REVIEW

File: components/TreeList.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import styled from "styled-components";
-import { allTrees } from "../resources/treeArray";
+import { allTrees } from "../resources/treearray";
```
Comment (mpagels) [inline components/TreeList.js:2]: is this working now?

File: pages/addpile/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  padding: 20px;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  color: black;
+  background-color: rgb(54, 89, 0);
+`;
+
+const StyledList = styled.ul`
+  list-style-type: disc;
+`;
```
Comment (mpagels) [inline pages/addpile/index.js:30]: Hey check here the semantic. You're using an `ul` element as a parent element. Semantic would be correct, that a `li` element is following. And you created a `ListItem` that you are not using.

File: pages/index.js [hunk 3]
```diff
@@ -1,18 +1,11 @@
...
 
 export default function HomePage() {
   return (
-    <div>
-      <Heading>Baumliste:</Heading>
-      <TreeList />
-    </div>
+    <>
+      <HeadingStart />
+      <AddPile />
```
Comment (mpagels) [inline pages/index.js:8]: You're using a page component inside a page component. That could be a problem.

Comment (mpagels) [reply pages/index.js:8]: Would recommend refactoring the `addPile` to the component folder.

File: pages/treelist/index.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  );
+};
+
+export default TreeList;
+
+const ListItem = styled.li`
+  text-align: left;
+  padding: 20px;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
```
Comment (mpagels) [inline pages/treelist/index.js:27]: Might be a good idea to make the font global available in the global styles to omit these lines here.

**Review by mpagels — CHANGES_REQUESTED** [review]
Hey Karl, very good start here.
Found a semantic issue. Pls review.

Keep up the good work!

**Comment by sebastianwel** [comment]
Hey Karl, reviewed all your changes and saw that you implemented Martins' changes correctly 🤘 Vercel looks also good, so nothing to add here from my side! 😁

**Review by sebastianwel — APPROVED** [review]

**Comment by vokarl** [comment]
Thanks for the review👋

**Pull request merged**